specs/_template/01_main.rst

```diff
@@ -0,0 +1,176 @@
+.. Getting Started
+.. Installation and IDE
+    To make your editing easier, we recommend to use VSCode editor with the extensions:
+    - reStructuredText (https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext)
+    - Table Formatter (https://marketplace.visualstudio.com/items?itemName=shuworks.vscode-table-formatter)
+.. The prerequisites for those extensions are python3 and doc8 packcages. Install instructions are here:
+    - https://docs.python-guide.org/starting/install3/osx/ (install python and pip, nothing else)
+    - After this, use pip3 to install other prerequisites below:
+    - https://docs.restructuredtext.net/articles/prerequisites.html
+.. Where to get more info
+    - http://docutils.sourceforge.net/docs/user/rst/quickref.html
+    - http://docutils.sourceforge.net/docs/user/rst/quickstart.html
+    - http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+    - http://docutils.sourceforge.net/docs/ref/rst/directives.html
+
+==========================================================
+SPECIFICATION_NAME
+==========================================================
+
+.. ==================== NOTE ===============================
+.. You can search and replace the EPIC, SPECIFICATION_NAME, 
+.. AUTHOR and CREATED_AT
+.. So that the header information is updated in this doc. 
+.. Do the same in other specification doc headers.
+.. =========================================================
+
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
```

Comment:
How about limiting the depth of the ToC? Makes it easier to read imo. I use
```
.. contents:: Table of Contents
    :depth: 3
```

Reply:
👍 I think a depth of 2 might also be enough but no strong opinion. It might change between specs too.